core/systems/input/input_manager.gd

```diff
@@ -158,14 +161,23 @@ func _input(event: InputEvent) -> void:
 		logger.debug("South released on its own.")
 		action_release(dbus_path, "ui_accept")
 
+	if event.is_action_pressed("ogui_north"):
+		logger.debug("North pressed on its own.")
+		action_press(dbus_path, "ogui_search")
+	elif event.is_action_released("ogui_north"):
+		logger.debug("North released on its own.")
+		action_release(dbus_path, "ogui_search")
+
 
 ## Handle guide button events and determine whether this is a guide action
 ## (e.g. guide + A to open the Quick Bar), or if it's just a normal guide button press.
 func _guide_input(event: InputEvent) -> void:
 	var dbus_path := event.get_meta("dbus_path", "") as String
 	# Only act on release events
 	if event.is_pressed():
-		logger.debug("Guide pressed. Waiting for additional events.")
+		logger.warn("Guide pressed. Waiting for additional events.")
+		# Set the gamepad profile to the global default so we can capture button events
+		launch_manager.set_gamepad_profile("")
```

Comment:
This ensures that we use the global profile and not the game's input profile for processing guide button combos and navigating the menu.